element_detector.py

```
@@ -75,8 +75,7 @@ def detect_and_annotate_elements(self, screenshot_path, browser_automation=None)
             
             return annotated_path
             
-        except Exception as e:
-            print(f"Error in element detection: {str(e)}")
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)